intake_bluesky/jsonl.py

```diff
@@ -4,15 +4,53 @@
 import pathlib
 
 from .in_memory import BlueskyInMemoryCatalog
+from .core import tail
 
 
 def gen(filename):
+    """
+    A JSONL file generator.
+
+    Parameters
+    ----------
+    filename: str
+        JSONL file to load.
+    """
     with open(filename, 'r') as file:
         for line in file:
             name, doc = json.loads(line)
             yield (name, doc)
 
 
+def get_stop(filename):
+    """
+    Returns the stop_doc of a Bluesky JSONL file.
+
+    The stop_doc is always the last line of the file.
+
+    Parameters
+    ----------
+    filename: str
+        JSONL file to load.
+    Returns
+    -------
+    stop_doc: dict
+        A Bluesky run stop document.
+    """
+    stop_doc = None
+    lastline = list(tail(filename))[0]
```

Comment:
This is marginally more efficient (avoids creating a `list`object).
```suggestion
    lastline, = tail(filename))
```